### ISSUE TITLE
Comparison to groupwise-ics

### ISSUE THREAD

**Issue by aspiers** [vuntz/maildir-to-ics#9]
Any idea how this is different to https://github.com/cbosdo/groupwise-ics?

(Same question in https://github.com/cbosdo/groupwise-ics/issues/2)

**Comment by vuntz** [comment]
@aspiers both are equally broken in the end ;-)

More seriously: I think @cbosdo's code is connecting directly to imap, while this code is using a local directory that is sync'ed with, say, offlineimap. Other than that, well, I'm not sure, I never really looked at @cbosdo's script.

**Comment by aspiers** [comment]
Hmm, they appear to [use SOAP](https://github.com/cbosdo/groupwise-ics/blob/master/connection.py).

**Comment by cbosdo** [comment]
the SOAP connection is a code piece towards uploading changes to GW since I can't do this using mails. Fetching the events is using IMAP

**Comment by aspiers** [comment]
> the SOAP connection is a code piece towards uploading changes to GW since I can't do this using mails

Ahh... so that means that maildir-to-ics will never be able to do bidirectional syncing.

**Comment by cbosdo** [comment]
> Ahh... so that means that maildir-to-ics will never be able to do bidirectional syncing.

Unless your post-office is SOAP-enabled.

**Comment by aspiers** [comment]
Not really, because IIRC the design of maildir-to-ics is only to work with local maildirs, not to interact with servers.

**Comment by vuntz** [comment]
@aspiers right. It's really a read-only solution.

**Comment by aspiers** [comment]
OK, so the only thing left to do here is clarify this info in the README.
